saleor/graphql/webhook/enums.py

```diff
@@ -41,7 +41,9 @@
     WebhookEventAsyncType.ACCOUNT_CHANGE_EMAIL_REQUESTED: (
         "Account email change is requested."
     ),
+    WebhookEventAsyncType.ACCOUNT_CONFIRMED: "An account is confirmed." + ADDED_IN_315,
```

Comment:
The label should be only in webhook description

Reply:
you mean the version label? if so maybe it is worth to clean  enums file. I saw that sometimes it was added but sometimes not